### PR TITLE
Add component to play audio stream in an audio element

### PR DESCRIPTION
**Pull request by kevadsett**
Also adds frontend controls for stream playback volume.

Feature is disabled in production build - edit environment.prod.ts to turn it on.

Assumes stream url is http://[hostname]:8000/bamp

### CODE REVIEW

**Comment by kevadsett** [comment]
Tested on Chrome 89 and Firefox 87 on MacOS 10.13

**Comment by chiguire** [comment]
It seems to work on Firefox, but you need to click the page settings and set it to Allow Audio, as it will block autoplay (this is something that the Join Stream button might have helped as the browser is more permissive with a user interaction)

Also, changing this settings needs a rebuild to become effective. I'll try this on production BAMP next to see if the stream works in the page.

File: frontend/BossAlienMediaPlayer/src/app/icecast-stream/icecast-stream.component.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  public isStreaming: boolean;
+  public currentVolume: number;
+
+  ngOnInit(): void {
+    this.isStreaming = environment.includeAudioStream;
+    if (this.isStreaming == false)
+    {
+      return;
+    }
+    this.streamSource = "http://" + window.location.hostname + ":8000/bamp"
```
Comment (chiguire) [inline frontend/BossAlienMediaPlayer/src/app/icecast-stream/icecast-stream.component.ts:28]: Could you change this URL to ```this.streamSource = "https://" + window.location.hostname + "/icecast/bamp"```? This should fix problems for all browsers

Comment (kevadsett) [reply frontend/BossAlienMediaPlayer/src/app/icecast-stream/icecast-stream.component.ts]: This is addressed in commit 71eb941

**Pull request merged**